src/lab2-tests.md

```diff
@@ -84,71 +124,12 @@ Layout also supports breakpoints after each addition to the display list:
 
     >>> test.unpatch_breakpoint()
 
-Testing `Browser`
------------------
-
-The Browser class defines a simple web browser, with methods to load,
-draw to the screen, and scroll down.
-
-Testing `Browser.load`
-----------------------
+Now let's test this, integrated into the full `Browser` class. Let's first mock a URL to load:
 
-Let's first mock a URL to load:
-
-    >>> url = 'http://test.test/example1'
-    >>> test.socket.respond(url, b"HTTP/1.0 200 OK\r\n" +
-    ... b"Header1: Value1\r\n\r\n" +
-    ... b"Body text")
+    >>> url = test.socket.serve("Body text")
 
 Loading that URL results in a display list:
 
-    >>> browser = lab2.Browser()
-    >>> browser.load(lab2.URL(url))
-    >>> browser.display_list
-    [(1, 1, 'B'), (2, 1, 'o'), (3, 1, 'd'), (4, 1, 'y'), (5, 1, ' '), (6, 1, 't'), (7, 1, 'e'), (8, 1, 'x'), (9, 1, 't')]
-
-And with breakpoints:
```

Comment:
What happened to the breakpoint tests?

Reply:
Oops, these got lost. I put them back.

Probably a separate question is to ask whether we really want to test these breakpoint tests, since they're not useful for anyone other than us; readers probably aren't going to try to figure out how to put breakpoints in the same place. Maybe we could have a separate tests file for them. But that's a different PR.